docs/rules/security/compiler-version.md

```diff
@@ -27,7 +27,7 @@ This rule accepts an array of options:
 ```json
 {
   "rules": {
-    "compiler-version": ["error","^0.8.14"]
+    "compiler-version": ["error","^0.8.23"]
```

Comment:
for future reference this is not really necessary, as it is only an example of how to configure it, but doesn't hurt either